src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -644,6 +645,10 @@ public function mustRevalidate()
      */
     public function getDate()
     {
+        if (!$this->headers->has('Date')) {
+            $this->setDate(new \DateTime(null, new \DateTimeZone('UTC')));
+        }
+
         return $this->headers->getDate('Date', new \DateTime());
```

Comment:
there is no point in adding a default here since it cannot trigger anymore. also this default could have the wrong timezone anyway.